src/utils/viewer/viewer.cpp

```diff
@@ -1048,6 +1078,25 @@ void Viewer::keyPressEvent(KeyEvent& event) {
     default:
       break;
   }
+
+  // Update map of moving/looking keys which are currently pressed
+  for (const auto& [k, b] : keysPressed) {
+    if (key == k) {
+      keysPressed[key] = true;
+    }
+  }
```

Comment:
There will only ever be one key pressed or released per call to keyPressEvent or keyReleaseEvent, so the loops should not be needed.

Reply:
use 
`if (keyPressed.count(key) > 0)`
instead of the loop and if check.